test/canary/run_canary_test.sh

```diff
@@ -11,6 +11,17 @@ if [ "$CLOUD_LABEL" != "Amazon" ]; then
     exit 0
 fi
 
+# https://stackoverflow.com/questions/16989598/comparing-php-version-numbers-using-bash/24067243#24067243
+function version_gt() { test "$(printf '%s\n' "$@" | sort -V | head -n 1)" != "$1"; }
+OPENSHIFT_VER=$(kubectl get managedcluster local-cluster -o jsonpath='{.metadata.labels.openshiftVersion}')
```

Comment:
@mikeshng You could have used label `openshiftVersion-major-minor`

Reply:
Updated PR as suggested by CICD to pass in these values.